src/nurse/views.py

```diff
@@ -18,6 +22,64 @@
 )
 
 
+class SendEmailToDoctorView(APIView):
+    def post(self, request, pk):
+        prescription = get_object_or_404(Prescription, id=pk)
+
+        patient = prescription.patient
+        if not patient:
+            return Response(
+                {"error": "Patient not found"}, status=status.HTTP_404_NOT_FOUND
```

Comment:
Maybe be more specific to avoid confusion e.g. "This prescription has no patient"

Reply:
Fixed